tailcfg/tailcfg.go

```diff
@@ -223,6 +224,12 @@ type Hostinfo struct {
 	RoutableIPs   []wgcfg.CIDR `json:",omitempty"` // set of IP ranges this client can route
 	Services      []Service    `json:",omitempty"` // services advertised by this machine
 
+	// Network conditions:
```

Comment:
I'd prefer a struct for all the network-related fields.

Reply:
Done.